Sources/PrivoSDK/internal/PrivoInternal.swift

```diff
@@ -10,7 +10,7 @@ internal class PrivoInternal {
     
     //MARK: - Static properties
         
-    static let rest = Rest()
+    static let rest: IRest = Rest.shared
```

Comment:
do not like singletons patter. It's really outdated

Reply:
I absolutely agree with you, but this line only adds a protocol to an already existing solution. I think we will make an effort to get away from all singletons in the future.